instrumentation/opentelemetry-instrumentation-httpx/src/opentelemetry/instrumentation/httpx/__init__.py

```diff
@@ -938,8 +938,8 @@ async def _handle_async_request_wrapper(  # pylint: disable=too-many-locals
 
         return response
 
+    @staticmethod
```

Comment:
Non-blocking comment:

wdyt of doing `@classmethod` instead and referring to fully qualified class name?

Reply:
Looks, good updated PR